ddtrace/llmobs/_llmobs.py

```diff
@@ -1307,27 +1315,37 @@ def submit_evaluation_for(
 
             if not isinstance(timestamp_ms, int) or timestamp_ms < 0:
                 error = "invalid_timestamp"
-                raise ValueError("timestamp_ms must be a non-negative integer. Evaluation metric data will not be sent")
+                raise ValueError("timestamp_ms must be a non-negative integer.")
 
             if not label:
                 error = "invalid_metric_label"
                 raise ValueError("label must be the specified name of the evaluation metric.")
 
-            metric_type = metric_type.lower()
-            if metric_type not in ("categorical", "score"):
+            if not metric_type or metric_type.lower() not in ("categorical", "numerical", "score", "boolean"):
                 error = "invalid_metric_type"
-                raise ValueError("metric_type must be one of 'categorical' or 'score'.")
+                raise ValueError("metric_type must be one of 'categorical', 'numerical', 'score', or 'boolean'.")
+
+            metric_type = metric_type.lower()
+            if metric_type == "numerical":
+                log.warning(
+                    "The evaluation metric type 'numerical' is unsupported. Use 'score' instead. "
+                    "Converting `numerical` metric to `score` type."
+                )
+                metric_type = "score"
```

Comment:
see above comment, we don't need to convert numerical to score metrics in this method